Scripts/Runtime/Entities/Util/WorldUtil.cs

```diff
@@ -0,0 +1,230 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Anvil.CSharp.Logging;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.LowLevel;
+using UnityEngine.PlayerLoop;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    /// <summary>
+    /// A system group to house a <see cref="World"/>s <see cref="EndInitializationEntityCommandBufferSystem"/>.
+    /// Used for mutli-world playerloop optimization.
+    /// </summary>
+    /// <remarks>Kept outside <see cref="WorldUtil"/> so their names read better in the editor.</remarks>
+    [DisableAutoCreation]
+    internal class EndInitializationCommandBufferSystemGroup : ComponentSystemGroup { }
+
+    /// <summary>
+    /// A system group to house a <see cref="World"/>s <see cref="EndSimulationEntityCommandBufferSystem"/>.
+    /// Used for mutli-world playerloop optimization.
+    /// </summary>
+    /// <remarks>Kept outside <see cref="WorldUtil"/> so their names read better in the editor.</remarks>
+    [DisableAutoCreation]
+    internal class EndSimulationCommandBufferSystemGroup : ComponentSystemGroup { }
+
+    /// <summary>
+    /// A <see cref="PlayerLoop"/> phase inserted immediately after <see cref="Update"/>.
+    /// Add via <see cref="WorldUtil.AddCustomPhasesToCurrentPlayerLoop" />.
+    /// </summary>
+    internal class PostUpdate
+    {
+        /// <summary>
+        /// Provides a <see cref="PlayerLoopSystem" /> instance to to identify this phase.
+        /// </summary>
+        public static PlayerLoopSystem PlayerLoopSystem
```

Comment:
Similar naming question if we should denote between what Unity gives you by default and what we've added custom.

Reply:
addressed